tests/e2e/fixtures/storage_fsx_dependencies.py

```diff
@@ -51,179 +42,63 @@ def get_fsx_mount_name(fsx_client, file_system_id):
-        vpc_id = response["cluster"]["resourcesVpcConfig"]["vpcId"]
-        cluster_security_group = response["cluster"]["resourcesVpcConfig"][
-            "clusterSecurityGroupId"
-        ]
-        subnet_id = response["cluster"]["resourcesVpcConfig"]["subnetIds"][0]
-
-        # Create Security Group
-        security_group_name = rand_name("fsx-security-group-")
-        response = ec2_client.create_security_group(
-            VpcId=vpc_id,
-            GroupName=security_group_name,
-            Description="My fsx security group",
-        )
-        fsx_volume["security_group_id"] = security_group_id = response["GroupId"]
-
-        # Open Port for CIDR Range
-        ec2_client.authorize_security_group_ingress(
-            GroupId=security_group_id,
-            IpPermissions=[
-                {
-                    "FromPort": 988,
-                    "ToPort": 988,
-                    "IpProtocol": "tcp",
-                    "UserIdGroupPairs": [{"GroupId": security_group_id}],
-                },
-            ],
-        )
-
-        ec2_client.authorize_security_group_ingress(
-            GroupId=security_group_id,
-            IpPermissions=[
-                {
-                    "FromPort": 988,
-                    "ToPort": 988,
-                    "IpProtocol": "tcp",
-                    "UserIdGroupPairs": [{"GroupId": cluster_security_group}],
-                },
-            ],
+        # Run the automated script to create the EFS Filesystem and the SC
+        fsx_auto_script_absolute_filepath = os.path.join(
+            os.path.abspath(sys.path[0]), "../" + fsx_auto_script_filepath
         )
 
-        # Create an Amazon fsx FileSystem for your EKS Cluster
-        response = fsx_client.create_file_system(
-            FileSystemType="LUSTRE",
-            SubnetIds=[subnet_id],
-            SecurityGroupIds=[security_group_id],
-            StorageCapacity=1200,
-            LustreConfiguration={"DeploymentType": "SCRATCH_2"},
-        )
-        file_system_id = response["FileSystem"]["FileSystemId"]
-
-        # Check for status of filesystem to be "available" before creating mount targets
-        wait_on_fsx_status("AVAILABLE", fsx_client, file_system_id)
-
-        # Write the FileSystemDetails to the metadata file
-        fsx_volume["file_system_id"] = file_system_id
-        fsx_volume["dns_name"] = get_fsx_dns_name(fsx_client, file_system_id)
-        fsx_volume["mount_name"] = get_fsx_mount_name(fsx_client, file_system_id)
-
-    def on_delete():
-        # Get FileSystem_ID
-        details_fsx_volume = metadata.get("fsx_volume") or fsx_volume
-        fs_id = details_fsx_volume["file_system_id"]
-        sg_id = details_fsx_volume["security_group_id"]
-
-        # Delete the Filesystem, does not provide a deleted status hence can't check delete status
-        fsx_client.delete_file_system(
-            FileSystemId=fs_id,
+        st = os.stat(fsx_auto_script_filepath)
```

Comment:
does st refer to stat?

Reply:
that was some bad variable naming, fixed it.